turnpike/plugins/x509.py

```diff
@@ -30,6 +30,11 @@ def headers_needed(self):
     def psk_check(self):
         """If HEADER_CERTAUTH_PSK is set in the config, then check that the
         request headers contain it and that its value matches the expected PSK."""
+
+        # When alternative gateway secret is created, turnpike provides this specific check
+        if self.cdn_psk in request.headers and request.headers[self.cdn_psk] == "alt-gateway-secret":
```

Comment:
do you think we could abstract `alt-gateway-secret` into a configurable variable via the app config and an env var? like how we do with `self.cdn_psk`